src/app/[affiliateSlug]/fairshareProvider.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import dynamic from "next/dynamic";
+import { env } from "@/../env";
+import { ReactNode } from "react";
+
+// Create a dynamic component that only loads on the client side
+const DynamicFairshareInitializer = dynamic(
```

Comment:
This is strange but the only way I could get it to work by circumventing all of the server stuff with a dynamic import